### DISCUSSION TITLE
ComfyUI Max retries exceeded with url: /object_info

### DISCUSSION THREAD

**Discussion by miguelgargallo** [open-webui/open-webui#5176] [General]
# Bug Report

## Installation Method

[Describe the method you used to install the project, e.g., git clone, Docker, pip, etc.]
With git clone

## Environment

- **Open WebUI Version:** latest
- **Ollama (if applicable):** latest

- Win11 Pro
- Latest Firefox

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

I try to connect comfyui from localhost, not docker, and here is the result

Something went wrong :/ HTTPConnectionPool(host='127.0.0.1', port=8188): Max retries exceeded with url: /object_info (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbf0b18b010>: Failed to establish a new connection: [Errno 111] Connection refused'))

![image](https://github.com/user-attachments/assets/b20f76de-ec97-4e72-92ee-ae7c29a3981c)

**Comment by fortarch** [comment]
same issue

**Reply by codedevmk** [reply]
To fix the issue, you need to make ComfyUI use your host's IP address instead of 127.0.0.1 (which is loopback). Since your Docker container is also running on 127.0.0.1, this causes a conflict.

Get Your Host IP: Run ipconfig /all on Windows and find your desired IP address.

Edit ComfyUI Startup:

Open your ComfyUI folder.
Locate either run_nvidia_gpu.bat or run_cpu.bat (depending on what you use).
Make a backup and edit the file to add --listen <your_host_ip> and --port 81 (replace <your_host_ip> with your actual IP).
Example command:
batch
Copy code
.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build --listen 10.1.0.51 --port 81
pause
Restart ComfyUI: Start ComfyUI, and then access it using your new IP (http://10.1.0.51:81).

Update Open WebUI:

Set the base ComfyUI URL to http://10.1.0.51:81 in your open web UI settings and save.
That's it—your container should now be able to connect properly.

**Reply by Zerta14** [reply]
thanks

**Reply by Da-Buche** [reply]
I know this is an old reply, but in case someone still have the same issue :

Instead of checking your computer IP using whatever script, there is a specific name for the host ip,
you can use it directly as ComfyUI Base URL (please replace the port number accordingly) :

http://host.docker.internal:8188

This might be safer and simpler to use 😊
